packages/suite/src/actions/suite/constants/messageSystemConstants.ts

```diff
@@ -0,0 +1,25 @@
+import { resolveStaticPath } from '@suite-utils/nextjs';
+
+export const FETCH_CONFIG_SUCCESS = '@message-system/fetch-config-success';
+export const FETCH_CONFIG_SUCCESS_UPDATE = '@message-system/fetch-config-success-update';
+export const FETCH_CONFIG_ERROR = '@message-system/fetch-config-error';
+
+export const SAVE_VALID_MESSAGES = '@message-system/save-valid-messages';
+export const DISMISS_MESSAGE = '@message-system/dismiss-message';
+
+// every 6 hours the message system config should be fetched
+export const FETCH_INTERVAL = 21600000; // in milliseconds
+// every 10 minutes the message system fetching interval should be checked
+export const FETCH_CHECK_INTERVAL = 600000; // in milliseconds
```

Comment:
Aaah now I see that we need to shorten the interval when the fetch fails. That makes sense.

I still find these two interval a bit more complicated. What about doing somethings like:

```js
function fetchConfig() {
   try {
     const res = fetchConfigFromServer()
     setTimeout(fetchConfig, 6 hours)
     return res
   } catch {
     setTimeout(fetchConfig, 10 minutes)
}
```

Feel free to keep it as is. RBY. (Resolve by yourself)

Reply:
The main reason for this, is possibility of inconsistent behavior of different browsers and operating system during the time when you make your computer sleep and wake up. Another can also be "pausing" tabs in the browser.

Using this 10 minutes interval it is guaranteed, that if you launch app, sleep your computer and wake it up after 6 hours, you will get new config in max 10 minutes.

Reply:
Ah, ok, got it. A comment briefly explaining this would be helful in the code. Something like:
```js
// Don't relly on 6 hour long JS intervals which can be flaky in various runtime environments.
```

Reply:
Added